ui/src/pages/Create.js

```diff
@@ -0,0 +1,645 @@
+i/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedHTMLMessage, FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+import Button from '@material/react-button';
+import Checkbox from '@material/react-checkbox';
+import MenuSurface, {Corner} from '@material/react-menu-surface';
+import Select from '@material/react-select';
+import Tab from '@material/react-tab';
+import TabBar from '@material/react-tab-bar';
+import TextField, {HelperText, Input} from '@material/react-text-field';
+
+import Footer from './../components/Footer.js';
+import LoadingIndicator from './../components/LoadingIndicator.js';
+import PFNotchedOutline from './../components/PFNotchedOutline.js';
+import RepoHeader from './../components/RepoHeader.js';
+
+const messages = defineMessages({
```

Comment:
Should have commented this earlier but shouldn't this be named MESSAGES because it's a constant?

Reply:
Yeah, true. Done for this file here; I'll fix the others in a separate PR.